starfish/test/full_pipelines/cli/test_iss.py

```diff
@@ -145,25 +144,13 @@ class TestWithIssData(unittest.TestCase):
                 "$['codebook']",
             ),
             "-o", lambda tempdir, *args, **kwargs: os.path.join(
-                tempdir, "results", "decoded-spots.nc"),
+                tempdir, "results", "spots.nc"),
```

Comment:
Previously we ran into some bugs where overwriting the spots file caused some serious problems. I would keep these having separate names from one another if possible.